csharp/src/Google.Protobuf.Test/JsonParserTest.cs

```diff
@@ -1,5 +1,5 @@
 ﻿#region Copyright notice and license
-// Protocol Buffers - Google's data interchange format
+// Protocol Buffers - Google\"s data interchange format
```

Comment:
Ha - no. I must have been doing a search and replace for " to \' in order to convert some JSON into a string literal. Will fix up as part of any other changes.